octobot_backtesting/collectors/data_collector.py

```diff
@@ -56,6 +58,12 @@ async def stop(self) -> None:
     async def start(self) -> None:
         raise NotImplementedError("Start is not implemented")
 
+    def is_in_progress(self):
+        return self.in_progress
+
+    def is_finished(self):
+        return self.finished
+
```

Comment:
Why adding getters while this attribute is public?

Reply:
just a mistake, normally it's private